pkg/kubelet/containerdshim/containerd_image.go

```diff
@@ -124,7 +124,24 @@ func pullImage(repo, tag string) error {
 	return nil
 }
 
-func createRootfs(repo, tag, path string) error {
+// image must be image reference.
```

Comment:
Would be nice if this could return a stat for the quantity pulled and/or estimated time remaining. So annoying to have things seemingly stop and hang while you wait for a large image to be pulled on a slow connection.